packages/v4/patternfly-docs/content/accessibility/progress/progress.md

```diff
@@ -0,0 +1,93 @@
+| `aria-describedby="[id of element that describes the progress]"` | `.pf-c-progress__bar` | Provides an accessible description for the progress component. |
+| `aria-valuetext="[loading state]"` | `.pf-c-progress__bar` |  Provide a text string that communicates current status. Only use if the important information about status is included in the text string. Do not use if percentage is the most important value to communicate. Some screen readers will ignore the percentage value determined from `aria-valuenow` when `aria-valuetext` is used. |
+| `aria-hidden="true"` | `.pf-c-progress__status` |  Hides the visible progress bar status from screen readers. This information is communicated by the aria attributes defined on the `.pf-c-progress__bar` element. |
+
+
+## Additional considerations
+
+Consumers must ensure they take any additional considerations when customizing a progress component, using it in a way not described or recommended by PatternFly, or in various other specific use-cases not outlined elsewhere on this page.
+
+### Aria-live
+
+When intending or expecting the progress to contain updates that dynamically appear or update, the recommended method of creating a live region is by using an `aria-live` value of "polite".
+
+If the progress component is intended or expected to contain time sensitive/critical information, `aria-live="assertive"` can instead be manually passed in. This should be used sparingly, as it will interrupt whatever task users are currently in the middle of.
+
+### Aria-atomic and aria-relevant
+
+You can further customize the way assistive technologies announce the contents of a live region via the `aria-atomic` and `aria-relevant` attributes.
+
+- `aria-atomic="true/false"` sets whether assistive technologies should announce the live region as a whole, even if only part of the region changes, ("true"), or if only the changes to the region should be announced ("false"). For example:
+
+  ```noLive
+    <div className="pf-screen-reader" aria-live="polite">
+      // render progress message
+      `Progress value is ${currentValue}%.`}
+    </div>
+    <Progress value={currentValue} title="Title" />
+  ```
+
+  In the above code block, if there were 2 progress updates already rendered within the live region and `aria-atomic="true"` was passed in, another update being rendered into the container would cause all 3 updates to be announced. If `aria-atomic="false"` were passed in instead, only the new update would be announced.
+
+- `aria-relevant` sets what types of changes are relevant to a live region and what qualifies as an update to it. The value passed in is a space-separated list of one or more of "additions", "removals", or "text", or "all" for all 3. The default value is "additions text" if there is no ancestor with this attribute passed in. An example for when a value of "removals" is passed in:
```

Comment:
Based on feedback from content, I think the commas and periods should go inside the quotes. 😄